bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -44,7 +44,7 @@ The following Things and OpenWebNet `WHOs` are supported:
 | Category                      |     WHO     |                       Thing Type IDs                       | Description                                                      | Status                                                                                                                                                                                                     |
 |-------------------------------|:-----------:|:----------------------------------------------------------:|------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Gateway Management            |    `13`     |                       `bus_gateway`                        | Any IP gateway supporting OpenWebNet protocol should work        | Successfully tested: F452, F453, F453AV,F454, F455, MyHOMEServer1, MyHOME_Screen10, MyHOME_Screen3,5, MH201, MH202, MH200N. Some connection stability issues/gateway resets reported with MH202            |
-| Lighting                      |     `1`     |             `bus_on_off_switch`, `bus_dimmer`              | BUS switches and dimmers                                         | Successfully tested: F411/2, F411/4, F411U2, F422, F429. Some discovery issues reported with F429 (DALI Dimmers)                                                                                           |
+| Lighting                      |     `1`     |             `bus_on_off_switch`, `bus_dimmer`, `bus_light_group`               | BUS switches and dimmers and groups of them                              | Successfully tested: F411/2, F411/4, F411U2, F422, F429. Some discovery issues reported with F429 (DALI Dimmers)                                                                                           |
```

Comment:
@lsiepel you mean I should reformat the WHOLE table here? 
This would mean to change all the lines in the table with this PR, loosing the focus on the modified lines related to the new Thing type...

Reply:
Yes, that is not a problem changes are highlighted. Just copy/paste it into a markdown formatter or use the one that comes with your ide.